components/datetime/src/scaffold/get_field.rs

```diff
@@ -4,9 +4,10 @@
 
 use icu_calendar::{
     types::{DayOfMonth, DayOfYearInfo, MonthInfo, Weekday, YearInfo},
-    AsCalendar, Calendar, Date, Iso,
+    AsCalendar, Calendar, Date,
 };
 use icu_time::{
+    provider::MinutesSinceEpoch,
```

Comment:
Not sure if you care about this or not but note that I am using `MinutesSinceEpoch`, which is defined in `provider`, in a non-provider module.

Reply:
Maybe move it into `scaffold`? I don't really want this to live in `zone`.